docker/Dockerfile.noefs

```diff
@@ -0,0 +1,18 @@
+FROM public.ecr.aws/seb-demo/magento:latest
```

Comment:
TODO - We need to change it to a private repo / bitnami official docker file.

Reply:
Yes: I'll flag that we need to have this image from a CI automated build pipeline : https://github.com/allamand/cdk-magento-fargate/issues/31